### PR TITLE
Add label snapshot_time for storage usage by user

### PR DESCRIPTION
**Pull request by Anbang-Hu**
```
storage_usage_in_bytes_by_user{exporter_name="storagemanager",instance="xxxxx:9094",job="serivce_exporter",mountpoint="xxx",scraped_from="storagemanager-l4bh9",snapshot_time="1591212391.827808",user="N/A",vc="xxx"}
```

### CODE REVIEW

File: src/StorageManager/storage_manager.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import time
 import os
 
+from datetime import datetime, timezone
```
Comment (xudifsd) [inline src/StorageManager/storage_manager.py:8]: `timezone` not used.

File: src/StorageManager/storage_manager.py [hunk 2]
```diff
@@ -100,7 +101,9 @@ def gen_usage_by_user_from_trees(self):
 
         usage_gauge = GaugeMetricFamily("storage_usage_in_bytes_by_user",
                                         "storage usage by each user",
-                                        labels=["vc", "mountpoint", "user"])
+                                        labels=["vc", "mountpoint", "user",
+                                                "snapshot_time"])
+        snapshot_time = str(datetime.timestamp(datetime.now()))
```
Comment (xudifsd) [inline src/StorageManager/storage_manager.py:106]: maybe `datetime.utcnow()`?

Comment (Anbang-Hu) [reply src/StorageManager/storage_manager.py:106]: good suggestion

File: src/StorageManager/storage_manager.py (hunk 1 above)
Comment (Anbang-Hu) [reply src/StorageManager/storage_manager.py]: removed

**Pull request merged**